File: tests/fixtures/compare/sample.zig
// Realistic Zig code with various issues
// TODO: handle errors properly

const std = @import("std");

pub fn process(items: []const i32) i32 {
    var total: i32 = 0;
    for (items) |item| {
        if (item > 10) {
            if (item > 20) {
                total += item;
            }
        }
    }
    return total;
}

pub fn empty() void {}

File: tests/fixtures/compare/sample.rb
# Realistic Ruby code with various issues
# TODO: refactor this module

def process(items)
  items.each do |item|
    puts item
    if item > 10
      if item > 20
        puts "big"
      end
    end
  end
end

def empty_method
end

def complex(a, b, c, d, e)
  a + b + c + d + e
end
